ZAPDTR/Makefile

```diff
@@ -121,6 +122,11 @@ build/%.o: %.cpp
 lib/libgfxd/libgfxd.a:
 	$(MAKE) -C lib/libgfxd
 
+.PHONY: StormLib
+StormLib:
+	LDFLAGS="" cmake -B ../StormLib/build -S ../StormLib
```

Comment:
We set `LDFLAGS=""` to avoid it using the existing LDFLAGS which wouldn't make sense here